GSoC-2025/movement:Calibrating confidence scores (Qianyi He).md

```diff
@@ -0,0 +1,83 @@
+## Movement: Calibrating Confidence Scores (Qianyi He)
+
+## Personal details
+- **Full name**: Qianyi He
+- **Email**: heqianyi926@uchicago.edu
+- **GitHub username**: Angelneer926
+- **Zulip username**: Qianyi He
+- **Location & time-zone**: Chicago, UTC-6
+- **Personal website / project portfolio**: Data Interaction https://github.com/Angelneer926/Data_interaction; Generative Model for Synthetic Histopathological Data Augmentation https://github.com/Angelneer926/Machine_Learning2/tree/main/Project
+- **Code contribution**
+https://github.com/neuroinformatics-unit/movement/pull/507
+
+- **Proposal discussion link**
```

Comment:
Can you make sure the final submission to GSoC does include the link to this PR here?